### ISSUE TITLE
No matching version found for (not latest version of) react-highcharts

### ISSUE THREAD

**Issue by romulo-rocha** [kirjs/react-highcharts#312]
If I try to install an older version of react-highcharts (react-highcharts@^10.0.2), npm says that no matching version is found.

> npm ERR! code ETARGET
npm ERR! notarget No matching version found for react-highcharts@10.0.2
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

I tried installing the latest version and it works perfectly, but the project I'm working on uses react-highcharts 10.0.2.
Is there any chance I can workaround on this?

npm version: 5.2.0
node version: 8.1.4

Thanks in advance.

**Comment by kirjs** [comment]
Hey Rômulo, here's the list of available versions
```bash
npm show react-highcharts@* version
react-highcharts@1.0.0 '1.0.0'
react-highcharts@0.0.1 '0.0.1'
react-highcharts@0.0.2 '0.0.2'
react-highcharts@0.0.3 '0.0.3'
react-highcharts@0.0.4 '0.0.4'
react-highcharts@0.0.5 '0.0.5'
react-highcharts@0.0.6 '0.0.6'
react-highcharts@0.0.7 '0.0.7'
react-highcharts@0.0.8 '0.0.8'
react-highcharts@0.0.9 '0.0.9'
react-highcharts@1.0.1 '1.0.1'
react-highcharts@1.0.2 '1.0.2'
react-highcharts@1.0.3 '1.0.3'
react-highcharts@2.0.0 '2.0.0'
react-highcharts@2.1.0 '2.1.0'
react-highcharts@3.0.0 '3.0.0'
react-highcharts@4.0.0 '4.0.0'
react-highcharts@5.0.0 '5.0.0'
react-highcharts@5.0.2 '5.0.2'
react-highcharts@5.0.3 '5.0.3'
react-highcharts@5.0.4 '5.0.4'
react-highcharts@5.0.6 '5.0.6'
react-highcharts@6.0.0 '6.0.0'
react-highcharts@6.0.1 '6.0.1'
react-highcharts@7.0.0 '7.0.0'
react-highcharts@8.0.0 '8.0.0'
react-highcharts@8.0.2 '8.0.2'
react-highcharts@8.0.3 '8.0.3'
react-highcharts@8.1.0 '8.1.0'
react-highcharts@8.1.1 '8.1.1'
react-highcharts@8.2.0 '8.2.0'
react-highcharts@8.3.0 '8.3.0'
react-highcharts@8.3.1 '8.3.1'
react-highcharts@8.3.2 '8.3.2'
react-highcharts@8.3.3 '8.3.3'
react-highcharts@8.4.0 '8.4.0'
react-highcharts@8.4.1 '8.4.1'
react-highcharts@8.4.2 '8.4.2'
react-highcharts@9.0.0 '9.0.0'
react-highcharts@10.0.0 '10.0.0'
react-highcharts@11.0.0 '11.0.0'
react-highcharts@11.5.0 '11.5.0'
react-highcharts@12.0.0 '12.0.0'
```

**Comment by romulo-rocha** [comment]
@kirjs Thank you so much for your reply!

So there's no way I can get react-highcharts@10.0.2 via npm install? Isn't it possible to include this version in the list of available versions?

**Comment by kirjs** [comment]
This version never existed, why would you want to have this specific version?

**Comment by romulo-rocha** [comment]
@kirjs It was just included on the package.json file of the project I'm working on. I'll fix it to 10.0.0 version.

Thanks for your support!

**Comment by kirjs** [comment]
Closing and there has been no activity in this issue for a while.
if this is still relevant, please feel free to reopen
